Makefile

```diff
@@ -32,9 +39,15 @@ ta:
 			  $@
 
 .PHONY: clean
+ifneq ($(wildcard $(TA_DEV_KIT_DIR)/host_include/conf.mk),)
 clean:
 	$(q)$(MAKE) -C host/xtest O=$(out-dir)/xtest q=$(q) $@
 	$(q)$(MAKE) -C ta O=$(out-dir)/ta q=$(q) $@
+else
+clean:
+	$(q)echo "TA_DEV_KIT_DIR is not correctly defined - Remove $(out-dir)"
+	$(q)rm -rf $(out-dir)
```

Comment:
:-)

Well, I'd much rather do an accidental `rm -rf /ta` than `rm -rf /`, but who can predict what people may store in `/ta` on their desktop machine?

I'm still opposed to this `rm -rf` especially since it's a makefile I can't avoid.